CHANGELOG.md

```diff
@@ -1,3 +1,79 @@
-v2.0
+# Changelog
 
-- Major Project Split. Please see the main ExAws repository for previous changelogs.
+## Unreleased
+
+- Add support for empty string/binary attributes
+- Empty string is saved as a space when using `put_item`
+
+## v4.0.0 - 2020-07-06
+
+- Empty string support
+
+## v3.0.3 - 2020-07-03)
```

Comment:
Could you remove the extra ")" from the end of the date?

Reply:
> Could you remove the extra ")" from the end of the date?

Done.